File: ViewModelsLayer/ViewModels/CardViewModels/ResponseCardViewModel.cs
using EntitiesLayer.Enums;

namespace ViewModelsLayer.ViewModels.CardViewModels
{
    public class ResponseCardViewModel
    {
        public SuitType Suit { get; set; }
        public int CardValue { get; set; }
        public CardNameType CardName { get; set; }
    }
}

File: API/Angular/src/app/viewmodels/CardViewModels/response-card-view-model.ts
import { Suit } from 'src/app/shared/enums/suit';
import { CardName } from 'src/app/shared/enums/card-name';

export class ResponseCardViewModel
{
  constructor(suit, cardValue, cardName)
  {
    this.suit = suit;
    this.cardValue = cardValue;
    this.cardName = cardName;
  }

  public suit: Suit;
  public cardValue: number;
  public cardName: CardName;
}
